webapp/src/components/EditAccountSettingsModal.vue

```diff
@@ -103,6 +111,21 @@ export default {
     modelValue: Boolean,
   },
   emits: ["update:modelValue"],
+  computed: {
+    displayNameValidationMessage() {
+      if (this.user.display_name && this.user.display_name.length > 150) {
+        return "Name should be less than 150 characters.";
+      }
+      return "";
+    },
```

Comment:
We should double check how this looks to the user when they put in something that fails at the API level